src/pages/patientView/clinicalTrialMatch/utils/location/CoordinateList.ts

```diff
@@ -0,0 +1,33 @@
+import { Coordinate } from './Coordinate';
+
+export const CITIES_AND_COORDINATES: { [name: string]: Coordinate } = {
+    Erlangen: new Coordinate(49.6, 11.0),
+    Leipzig: new Coordinate(51.359, 12.377),
+    Dresden: new Coordinate(51.05, 13.739),
+    Würzburg: new Coordinate(49.48, 9.56),

```

Comment:
There is probably a better way to convert city names into geographic coordinates.